src/extensions/replay/sessionrecording.ts

```diff
@@ -909,9 +928,9 @@ export class SessionRecording {
             blockSelector: undefined,
             ignoreClass: 'ph-ignore-input',
             maskTextClass: 'ph-mask',
-            maskTextSelector: undefined,
+            maskTextSelector: this.masking.maskTextSelector ?? undefined,
             maskTextFn: undefined,
-            maskAllInputs: true,
+            maskAllInputs: this.masking.maskAllInputs,
```

Comment:
i changed with this while trying to figure things out and didn't bother to change it back
feel free to change it back if you prefer the previous style i was just moving things around :)

Reply:
nope, this was the lazy change that kept growing so I changed it back 🙈